index.html

```diff
@@ -45,6 +45,178 @@ <h1>
       <img src="images/Normal Button/Secondary/Disabled.png" alt="">
     </section>
   </section>
+  <section class="r_works">
+    <section class="title">
+      <p>My Recent Works</p>
+      <p class="s"></p>
+    </section>
+    <!-- <div>1</div> -->
+    <section class="works grid-container">
+        <section class="grid_1-item">
+          <!-- <img src="#" alt="" class="img_place_holder"> -->
```

Comment:
- [ ] Kindly remove unnecessary comments from the codebase to increase the readability of the code.

Reply:
Ok Done

Reply:
@Tufahel Thanks a lot. I hope everything is fine now